src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
Let's specify what exactly the result is
![image](https://github.com/mate-academy/js_style-to-object/assets/96409563/5e92a906-e9f9-4ef7-89f6-9b7431f46c28)

Reply:
Ok, i pay attention on this in future works. Thanks 🤝